stencil-library/src/components/form/form-control-text.tsx

```diff
@@ -1,24 +1,43 @@
-import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+import { Component, Host, h, Prop, Event, EventEmitter, State, Element, Watch } from '@stencil/core';
 
 @Component({
   tag: 'form-control-text',
   styleUrl: 'form-control-text.scss',
   shadow: true,
 })
 export class TextInput {
+  @Element() el: HTMLElement;
+
   @Prop() label: string;
   @Prop() name: any;
   @Prop() error: string;
   @Prop() defaultValue: string;
+  @State() input: string;
   @Prop() inputHandler: (name: string, value: string) => void;
   @Event() formControlInput: EventEmitter<any>;
   @Event() formControlBlur: EventEmitter<any>;
 
+  @Watch('defaultValue')
+  handleDefaultValueChange(newValue: string) {
```

Comment:
Have you tried putting a console log in here to see how often this runs? It seems like it would run every time the `value` changes instead of just once when it gets the default value the first time

Reply:
Very good catch. Fixed that!